mqtt/mqtt-bridge/src/client.rs

```diff
@@ -158,8 +280,10 @@ impl<T: EventHandler> MqttClient<T> {
     pub async fn handle_events(mut self) -> Result<(), ClientConnectError> {
         while let Some(event) = self.client.try_next().await.unwrap_or_else(|e| {
             error!(message = "failed to poll events", error=%e);
+            // TODO: handle the error and recreat the client/bridge
             None
         }) {
+            debug!("handle event {:?}", event);
```

Comment:
Ok with the Event type and then each property. Probably I bit to much to log every message,  it might be good for initial testing, then I can change and just log message received.